accounts/views.py

```diff
@@ -1,12 +1,25 @@
+import json
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth import authenticate, login, logout
 from django.http import HttpResponse, JsonResponse
 from django.db import transaction
-
-from .forms import CustomUserCreationForm
+from recommendations.models import (
```

Comment:
ain't no way this needs to be three lines - sometimes it won't format back if you reduce the number of imports from there
so yeah you should be able to replace it with just 'from recommendations.models import Movie'